static/styles/left_sidebar.css

```diff
@@ -62,6 +62,52 @@ li.show-more-topics {
     }
 }
 
+#show_more_pms {
```

Comment:
I think this should probably have a name more like `li.show-more-private-conversations`, for similarity to `show-more-topics`.

Reply:
Also, could we share CSS with `show-more-topics`, rather than copying it?

Reply:
Hmm -- would fix this.